lib/public/views/Runs/ActiveColumns/createInelasticInteractionRateActiveColumns.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+import { formatFloat } from '../../../utilities/formatting/formatFloat.js';
+
+const GREEK_LOWER_MU_ENCODING = '\u03BC';
+
+/**
+ * Factory for inelastic interaction rate values active columns configuration
+ *
+ * @param {Run[]} runs list
+ * @return {object} active columns configuration
+ */
+export const createInelasticInteractionRateActiveColumns = (runs) => ({
+    muInelasticInteractionRate: {
+        name: `${GREEK_LOWER_MU_ENCODING}(INEL)`,
+        visible: runs.some((run) => run.pdpBeamType === 'pp'),
```

Comment:
You removed the notion of function for visible, and in any case I think it is very strange to have a table with columns depending on the data. If your data have pp on some page and pbpb in another, you'll have changing columns when you change page, which is very wrong. 
What I would do here is to have 2 activeColumns definitions:

- `inelasticInteractionRateActiveColumnsForPp`
- `inelasticInteractionRateActiveColumnsForPbpb`

And then depending on where you display it, either you display both with `-` when it do not apply, or either you display both but only if it do not rely on the runs list being displayed